Proposed changes Sectioned Shapes (#93)
bSI-InfraRoom/IFC-Specification: IFC4x3/Sections/Resource definition data schemas/Schemas/IfcGeometricModelResource/Entities/IfcSectionedSurface/Documentation.md

> A surface constructed by sweeping potentially varying open cross sections along a curve horizontally (or near horizontally). The surface is generated by sweeping the CrossSections between CrossSectionPositions; linear interpolation is assumed, unless transitions curves between cross section points are indicated by OpenCrossProfileDef.Tags.

Reviewer: ```suggestion
A surface constructed by sweeping potentially varying open cross sections along a curve horizontally (or near horizontally). The surface is generated by sweeping the _CrossSections_ between _CrossSectionPositions_; linear interpolation is assumed, unless transitions curves between cross section points are indicated by OpenCrossProfileDef.Tags.
```

> The profile normal is derived from the associated _IfcAxis2PlacementLinear_, not necessarily the tangent of the _Directrix_. The profile X axis is oriented perpendicularly to the left of the _Directrix_ (same direction as _RefDirection_ derived in _IfcAxis2PlacementLinear_) as facing forward along the directrix, and the profile Y axis is oriented upwards according to _Axis_ attribute of _IfcAxis2PlacementLinear_.

Reviewer 2: This up, left etc. is problematic. I would propose to avoid it entirely.
```suggestion
The profile normal is derived from the associated _IfcAxis2PlacementLinear_, not necessarily the tangent of the _Directrix_. The profile Y axis is oriented according to _Axis_ attribute of _IfcAxis2PlacementLinear_ and the X is derived from the cross product of _Directrix_ and the _Axis_ attribute of _IfcAxis2PlacementLinear_.
```

Reviewer 3: I agree with removal of up left etc.

However, I wonder if it's not better to use Axis1 Axis2 and Axis3 as defined by the IfcAxis2Placements?

Figure 1 illustrates an _IfcSectionedSurface_ defined using _IfcOpenCrossProfileDef_

!["sectioned surfacequot;](../../../../../../figures/IfcSectionedSurface_1.png "Figure 1 &mdash; Sectioned surface using _IfcOpenCrossProfileDef_")

> In case of branching longitudinal breaklines, the SectionedSurface may use OpenCrossProfileDef instances with varying number of cross section points. In that case, the point in two consecutive cross sections that are connected are identified by the same tag value.

Reviewer: ```suggestion
In case of branching longitudinal breaklines, the _SectionedSurface_ may use _OpenCrossProfileDef_ instances with varying number of cross section points. In that case, the point in two consecutive cross sections that are connected are identified by the same tag value.
```

> In case of branching longitudinal breaklines, the _SectionedSurface_ may use _OpenCrossProfileDef_ instances with varying number of cross section points. In that case, the point in two consecutive cross sections that are connected are identified by the same tag value.

Reviewer 2: We need stronger wording here. Two questions:
1. Would a WR make sense? Restricting the profile type.
2. Would we allow `IfcArbitraryOpenCrossProfileDef` with `IfcCartesianPointList` geometry?
```suggestion
In case of branching longitudinal breaklines, the _IfcSectionedSurface_ shall use _IfcOpenCrossProfileDef_ instances with varying number of cross section points. In that case, the point in two consecutive cross sections that are connected are identified by the same tag value.
```

Reviewer: Solution:
Place in General usage the restrictions on profiles and Cartesian Point lists for Tags.

Reviewer 3: Agree to move this to the usages.
